src/component/modal/ModalInfo.svelte

```diff
@@ -14,11 +14,24 @@
 	let contributor: Icontributor[] = [];
 	// let changelog: string;
 	onMount(async () => {
-		contributor = (
-			await fetch('https://api.github.com/repos/kodepandai/colorful-quran/contributors').then(
-				(data) => data.json()
-			)
-		).filter((c) => !c.login.includes('bot'));
+		// contributor = (
+		// 	await fetch('https://api.github.com/repos/kodepandai/colorful-quran/contributors').then(
+		// 		(data) => data.json()
+		// 	)
+		// ).filter((c) => !c.login.includes('bot'));
+		contributor = await fetch("https://api.github.com/repos/kodepandai/colorful-quran/contributors")
+			.then(res => res.json())
+			.then(contributor => {
+				const pending = contributor
+					.filter(c => !c.login.includes("bot"))
+					.map(async c => {
+						const res = await fetch(c.url)
+						const json = await res.json()
+						c.login = json.name
+						return c
+					})
+				return Promise.all(pending)
```

Comment:
Hi @falentio, can we stick to username instead of name? Because same github account doesn't have name. It will return null. Also we need add overflow-y-scroll to tab content when we change the display to name.
![image](https://user-images.githubusercontent.com/8775678/210167348-d12fb0df-183e-464d-851f-16d157b9e781.png)